pcdsdevices/tests/test_spectrometer.py

```diff
@@ -4,7 +4,7 @@
 from ophyd.sim import make_fake_device
 
 from ..spectrometer import Kmono, VonHamos4Crystal, VonHamosFE, VonHamosFER
-
+from ..spectrometer import HXRSpectrometer
```

Comment:
Combine these lines

Reply:
Combining those two lines results in a test fail due to flake8 errors.

Reply:
What are the errors?

Reply:
<img width="983" alt="Screen Shot 2022-03-03 at 12 58 22 AM" src="https://user-images.githubusercontent.com/35245246/156531352-e1c581c5-70aa-4676-bf26-4e9dc219ecab.png">

Reply:
Those can be fixed by splitting it into two lines with parentheses (e.g. see signal.py) and adding the second blank line back.